docs/PacketFence_MSPKI_Quick_Install_Guide.asciidoc

```diff
@@ -0,0 +1,360 @@
+
+This will give you the certificate (public key) for PacketFence.
+
+Now download the CA file by reaching the following URL in your browser: `http://<ServerDNSName>/CertSrv/`.
+
+Click `Download a CA certificate, certificate chain or CRL`, select your CA certificate in the list, select `Base 64` as the encoding method and finaly click `Download CA certificate`.
+
+Copy those files from the location you downloaded them to PacketFence.
+
+Step 2: Configuring PacketFence
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Certificate storage on PacketFence
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+ 
+It is recommended to create a separate directory to separate EAP-TLS certificates from server certificates:
+
+ # mkdir /usr/local/pf/conf/ssl/tls_certs/
+
+RADIUS EAP-TLS authentication requires three files, the CA certificate, the server certificate and the server private key.
+
+Copy those files in your newly created folder:
+
+* Private Key of the RADIUS server (obtained while generating the CSR)
+* Certificate for RADIUS (obtained from the submitted CSR)
+* CA Certificate (downloaded from the NDES website)
+
+Ensure that the files are readable by the user `pf`: 
+
+ # chown pf:pf /usr/local/pf/conf/ssl/tls_certs/*
+ 
+RADIUS EAP-TLS and MSPKI
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+In order to use the certificates generated by the MSPKI, edit the radius EAP configuration file.
+
+Edit `/usr/local/pf/conf/radiusd/eap.conf` and replace the following lines with references to your new certificates in the `tls` configuration block:
+
+ private_key_file = %%install_dir%%/conf/ssl/server.key
+ certificate_file = %%install_dir%%/conf/ssl/server.pem
+
+E.g.
+
+ private_key_file = %%install_dir%%/conf/ssl/tls_certs/server.key
+ certificate_file = %%install_dir%%/conf/ssl/tls_certs/server.pem
+ CA_file = %%install_dir%%/conf/ssl/tls_certs/MyCA.pem
+
+Certificate revocation checks have to be configured in the `OCSP` sub-block of `tls`.
+
+For example: 
+
+    OCSP {
+        enable = yes
+        override_cert_url = yes
+        url = "http://<MSPKI ServerDNSName or IP>/OCSP"
+    }
+
+Restart radiusd to regenerate the new configuration files and enable EAP-TLS using your CA signed certificates:
+
+ # /usr/local/pf/bin/pfcmd service radiusd restart
+
+PacketFence PKI provider configuration
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Using the PKI requires configuring the PKI providers section in the PacketFence GUI under `Configuration -> Users`. The provider configuration defines how PacketFence connects to the MSPKI and what information will be sent.
+
+Add a new PKI provider and select SCEP.
+
+Fill out the form for a PKI provider according to your Certificate of Authority configuration.
+
+image::docs/images/scep-ms-pki-provider.png[scaledwidth="100%",alt="MSPKI configuration"]
```

Comment:
The issue here is that there is no tools to remotely revoke a certificate on MSPKI. To do it we need to send the command to be executed on MSPKI (We might able by Wmi). Execute cmd -> command to revoke (working on it)